src/main/resources/assets/api-docs.yaml

```
@@ -3224,6 +3224,21 @@ paths:
           description: User not found
         500:
           description: Server error.
+  /api/user/institution/unassigned:
+    get:
+      summary: Find users that do not belong to an institution
+      description: Find users that do not belong to an institution
+      tags: 
+        - User
+      responses: 
+        200:
+          description: An array of users that do not belong to an institution
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/User'
```

nit, this should be an array of users, not a single user:
```suggestion
              schema:
                type: array
                items:
                  $ref: '#/components/schemas/User'
```